test/e2e/metrics_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
```

Comment:
Given the test has nothing to do with cluster autoscaling, this should be a separate PR